### DISCUSSION TITLE
How to replace Get.putAsync with the refreshed version ?

### DISCUSSION THREAD

**Discussion by sumitsharansatsangi** [Aniketkhote/refreshed#11] [Q&A] [answered]
I have used Get.putAsync , What is the equivalent of it in the Refreshed package?

**Comment by sumitsharansatsangi** [comment, marked as answer]
Author has added this functionality in ^2.5.0.

**Reply by Aniketkhote** [reply]
Refreshed is derived from the GetX5 branch, and it's unclear why putAsync was removed by Jonny Borges without a clear explanation or migration details provided. In light of this, I've taken the liberty to reintroduce putAsync functionality. However, please be aware that while I've restored it, there might be unforeseen issues or changes in behavior compared to its previous implementation. If you encounter any problems or have questions about its functionality, please don't hesitate to reach out.

**Reply by sumitsharansatsangi** [reply]
Sure, @Aniketkhote .
